include/odp/api/spec/schedule_types.h

```diff
@@ -260,6 +260,18 @@ typedef struct odp_schedule_config_t {
 	 */
 	uint32_t max_flow_id;
 
+	/** Predefined scheduling groups */
+	struct {
```

Comment:
Documentation could be a bit more explicit that this config is used to enable/disable predefined groups (e.g. 1 == enable, 0 == disable) and that default value is "enabled" for all.

Reply:
Fixed in V2.